src/types/profile.ts

```diff
@@ -300,3 +302,44 @@ export type Aggregations = {
   likely_region: string | null
   likely_region_score: number | null
 }
+
+export type ProfileList = {
+  id: number
+  profile_id: Profile['id']
+  name: string
+  description: string | null
+  meta: CustomFields | null
+  handle: string
+  sort_order: string | null
+  items: Array<ProfileListItem>
+  id_shared: boolean
+  shares: ProfileListShare[]
+  tags: Array<string>
+  custom_fields: CustomFields
+  definition: ListDefinition
+  custom_date: Date
+  location: Location
+  staff: Profile
+  created_at: Date
+  updated_at: Date
+}
+
+export type ProfileListShare = {
+  id: number
+  handle: string
+  profile_id: Profile['id']
+  product_list_id: List['id']
+  created_at: Date
+  updated_at: Date
+}
+
+export type ProfileListItem = { // TODO: Is this a generic list item?
```

Comment:
It's its own thing
https://github.com/omneo/api/blob/master/app/Http/Requests/StoreProductListItem.php